### PR TITLE
Fix Broken Dashboard - ReImplement Anomaly Dash

### LINKED ISSUE #1457: Update dashboard to look at new anomaly db tables
_Issue [JWQL-168](https://jira.stsci.edu/browse/JWQL-168) was created on JIRA by [Bryan Hilbert](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hilbert):_

The dashboard produces a plot showing the number of each type of anomalies in the database for each instrument. Since we switched the anomaly database into the Django models (djangomodeldb), the dashboard is now looking at the wrong location for the anomaly info. 

We should point the dashboard to the correct location. The jwqldb page also looks in the wrong place when asked to show an <instrument>_anomaly table, so we should make the same update there.

### PR DESCRIPTION
**Pull request by BradleySappington**
Fix broken dashboard due to references to unused anomaly tables
Re write anomaly dashboard functionality to use new table structure.

### CODE REVIEW

**Comment by york-stsci** [comment]
Is it worth changing any of the database queries that use database_interface to use the django models instead?

File: jwql/website/apps/jwql/bokeh_containers.py [hunk 1]
```diff
@@ -282,10 +282,10 @@ def generic_telemetry_plot(times, values, name, nominal_value=None, yellow_limit
     if nominal_value is not None:
         fig.line(times, np.repeat(nominal_value, len(times)), line_dash='dashed')
 
-    fig.xaxis.formatter = DatetimeTickFormatter(hours=["%d %b %H:%M"],
-                                                days=["%d %b %H:%M"],
-                                                months=["%d %b %Y %H:%M"],
-                                                years=["%d %b %Y"],
+    fig.xaxis.formatter = DatetimeTickFormatter(hours="%d %b %H:%M",
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/bokeh_containers.py:286]: How did this function ever work if DateTimeTickFormatter was never imported?

File: jwql/website/apps/jwql/bokeh_dashboard.py [hunk 2]
```diff
@@ -37,7 +37,7 @@
 
 from bokeh.layouts import column
 from bokeh.models import Axis, ColumnDataSource, DatetimeTickFormatter, HoverTool, OpenURL, TapTool
-from bokeh.models import TabPanel, Tabs
+from bokeh.models.layouts import TabPanel, Tabs
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/bokeh_dashboard.py:40]: In bokeh_containers.py these imports do not include "layouts"

**Review by bhilbert4 — APPROVED** [review]
I haven't tried looking at the dashboard yet, but LGTM code-wise.

File: jwql/website/apps/jwql/bokeh_containers.py (hunk 1 above)
Comment (BradleySappington) [reply jwql/website/apps/jwql/bokeh_containers.py:286]: This function never worked and is not called.  bokeh_containers.py is NOT in good shape, but this isn't the PR to fix it.

**Comment by bhilbert4** [comment]
The page looks about as good as it can get when running locally.

**Comment by BradleySappington** [comment]
> Is it worth changing any of the database queries that use database_interface to use the django models instead?

@york-stsci 
I considered this and chose not to just because they will continue working with their current setup and I wanted to get the dashboard functional again.  We can discuss if its worth the time commitment to have django do them, and if so, can have it be part of the epic of the migration to the model system.

File: jwql/website/apps/jwql/bokeh_dashboard.py (hunk 2 above)
Comment (BradleySappington) [reply jwql/website/apps/jwql/bokeh_dashboard.py:40]: TabPanel and Tabs now exists in layouts as of bokeh 3.0

**Comment by york-stsci** [comment]
Ultimately, at the end of the epic, the database_interface code will have been removed, so the answer for "is it worth the time commitment" is going to be "yes" eventually, but it certainly doesn't need to be a part of this PR specifically.

**Comment by BradleySappington** [comment]
Well that makes that decision easy :)

File: jwql/website/apps/jwql/bokeh_dashboard.py (hunk 2 above)
Comment (bhilbert4) [reply jwql/website/apps/jwql/bokeh_dashboard.py:40]: So then should the import statement in bokeh_containers.py be updated to include "layouts"?

Comment (BradleySappington) [reply jwql/website/apps/jwql/bokeh_dashboard.py:40]: 100% - Good catch

**Pull request merged**